lib/rex/proto/kerberos/model.rb

```diff
@@ -8,14 +8,17 @@ module Model
 
         # Application Message Id's
 
-        AS_REQ = 10
-        AS_REP = 11
-        TGS_REQ = 12
-        TGS_REP = 13
-        KRB_ERROR = 30
-        TICKET = 1
-        AUTHENTICATOR = 2
-        AP_REQ = 14
+        #module MessageType
```

Comment:
@smashery Sounds good to me to namespace this; Are we good to put this in a separate PR, or were there technical reasons for not wanting to add the namespace 👀

Reply:
Yeah mostly on this one this was a note for later given all the merging I knew we'd have to do - wanted to reduce conflicts. Might just do a separate PR later.

Reply:
That's a good call, thanks for the clarification! :+1: